usr/lib/api/policy.c

```diff
@@ -255,6 +255,7 @@ static CK_RV policy_extract_key_data(get_attr_val_f getattr, void *d,
         *comptarget = COMPARE_SYMMETRIC;
         break;
     case CKK_AES:
+    case CKK_AES_XTS:
         *siglen = 128;
```

Comment:
Are there signatures with AES_XTS?  If not, you should set `*siglen = 0` here.  I think AES-XTS needs an own case without fallthrough.